Fix:

../gst-plugins-base-1.24.5/gst-libs/gst/gl/egl/gstgldisplay_egl.c: In function 'gst_gl_display_egl_get_from_native':
| ../gst-plugins-base-1.24.5/gst-libs/gst/gl/egl/gstgldisplay_egl.c:241:39: error: 'EGL_PLATFORM_SURFACELESS_MESA' undeclared (first use in this function)
|   241 |     ret = _gst_eglGetPlatformDisplay (EGL_PLATFORM_SURFACELESS_MESA,
|       |    

diff --git a/gst-libs/gst/gl/egl/gstgldisplay_egl.c b/gst-libs/gst/gl/egl/gstgldisplay_egl-2.c
index e080580..78868a6 100644
--- a/gst-libs/gst/gl/egl/gstgldisplay_egl.c
+++ b/gst-libs/gst/gl/egl/gstgldisplay_eglc
@@ -62,6 +62,9 @@ GST_DEBUG_CATEGORY_STATIC (gst_gl_display_egl_debug);
 #ifndef EGL_PLATFORM_ANGLE_ANGLE
 #define EGL_PLATFORM_ANGLE_ANGLE 0x3202
 #endif
+#ifndef EGL_PLATFORM_SURFACELESS_MESA
+#define EGL_PLATFORM_SURFACELESS_MESA 0x31DD
+#endif
 
 typedef EGLDisplay (*_gst_eglGetPlatformDisplay_type) (EGLenum platform,
     void *native_display, const EGLint * attrib_list);
